### DISCUSSION TITLE
fpa reset

### DISCUSSION THREAD

**Discussion by LyxxSK** [jilleb/MQB-FPA#58] [Exploration and research]
hi, i have stock dataset for 3q0907530Q, and i have problem in stock dataset after car off fpa reset on latest set, but sample: when i selected sport fpa, and off, after start car fpa is still sport, but engine set to normal. i checked in measures in vcds when again klik on sport fpa , engine is setted on sport. but again car off, and start, everything is in sport but engine reset to normal. how edit please fpa for disable engine reset to normal please?

**Comment by olli991** [comment]
That is the normal behavior. This repo is exactly about on how you can change this... read into the Wiki and all and then you can fix that.
